src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -10,9 +13,9 @@
 
   <!-- Mobile and legacy targets - comment these out or set env variable MSAL_DESKTOP_ONLY_DEV to speedup the build -->
   <PropertyGroup Condition="'$(MSAL_DESKTOP_ONLY_DEV)' == ''">
-    <TargetFrameworkNetStandard>netstandard1.3</TargetFrameworkNetStandard>
+    <TargetFrameworkNetStandard>netstandard2.0</TargetFrameworkNetStandard>
     <TargetFrameworkNetDesktop45>net45</TargetFrameworkNetDesktop45>
-    <TargetFrameworkUap>uap10.0</TargetFrameworkUap>
+    <TargetFrameworkUap>uap10.0.17763</TargetFrameworkUap>
```

Comment:
So this will be a breaking change for projects using these targets?
netstandard13
.Net Core 1.x
.Net 4.6 (except this, Msal net45 will be used)
Mono 4.6
XamariniOS 10 (and this will use explicit iOS target)
Xamarin Mac 3
Xamarin Android 7
UAP 10

This has been approved by Bogdan but just mentioning for completeness.

Reply:
Yes.
@bgavrilMS FYI